### DISCUSSION TITLE
drag a sample

### DISCUSSION THREAD

**Discussion by tjingboem** [kunstmusik/blue#614] [Ideas]
As a Reaper user, i am accustomed to drag the side of a Media object, and then this Object goes into repetition mode.
So i was dropping a sample on the Audio Layer and started dragging the sample so it would repeat itself- nothing happened of course.
![Screenshot from 2021-09-01 11-42-44](https://user-images.githubusercontent.com/6670911/131650251-f72dd425-55c4-41b5-909d-c18503a35c9f.png)
Is this something to have in Blue?

**Comment by kunstmusik** [comment]
@tjingboem This makes sense to me and I'm surprised I hadn't implemented it already. Could you file an issue for this?

**Reply by tjingboem** [reply]
i filed the issue: drag a sample #615

Yes, the Repeat function is implemented for all SoundObjects, but notfor the Audio Files on the AudioLayer. It is now that i am using the AudioLayer i notice this.
